lib/draw.c

```diff
@@ -192,19 +281,30 @@ void projectMeshTo2D(mesh inputMesh, const double distance)
     }
 }
 
-void drawMeshOnScreen(mesh inputMesh, double origin[2], double ratio, int screen[MAX_X][MAX_Y]) 
+void drawMeshOnScreen(mesh inputMesh, double origin[2], screenStruct screen, vector *inputVecArr) 
```

Comment:
what is inputVecArr, this is a bit of a redundant name,  maybe call it normalVectors or something?